config/php-scoper/idiorm.inc.php

```diff
@@ -14,7 +14,7 @@
 	 * For more see: https://github.com/humbug/php-scoper#finders-and-paths
 	 */
 	'finders'                    => array(
-		Finder::create()->files()->in( 'vendor/j4mie/idiorm' ),
+		Finder::create()->files()->in( 'vendor/j4mie/idiorm' )->exclude( 'demo.php' )->name( [ '*.php', 'LICENSE', 'composer.json' ] ),
```

Comment:
The `demo.php` file still exists in the `vendor_prefixed`-folder after running the prefix-dependencies command. 

```
Finder::create()->files()->in( 'vendor/j4mie/idiorm' )->name( [ 'idiorm.php', 'LICENSE', 'composer.json' ] ),
```

This command seems to work though, not sure why the exclude is not working for me.